src/main/java/com/google/devtools/build/lib/packages/semantics/BuildLanguageOptions.java

```diff
@@ -770,6 +770,14 @@ public final class BuildLanguageOptions extends OptionsBase {
               + " number of files is not 1.")
   public boolean incompatibleLocationsPrefersExecutable;
 
+  @Option(
+      name = "internal_starlark_utf_8_byte_strings",
+      defaultValue = "true",
+      documentationCategory = OptionDocumentationCategory.UNDOCUMENTED,
```

Comment:
Fine to leave undocumented for users, but please add a docstring for our own benefit.